src/Geta.NotFoundHandler/Core/Redirects/RedirectsCsvParser.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Geta Digital. All rights reserved.
+// Functionality added by Jacob Spencer 06/2024
+// Licensed under Apache-2.0. See the LICENSE file in the project root for more information
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using CsvHelper;
+using Geta.NotFoundHandler.Models;
+using Microsoft.Extensions.Logging;
+
+namespace Geta.NotFoundHandler.Core.Redirects;
+
+public class RedirectsCsvParser
```

Comment:
Add IRedirectsParser interface with `CustomRedirectCollection LoadFromStream(Stream csvContent)` as a method and implement this interface here. Also, add implementation for existing RedirectsXmlParser.